src/TypeUtils.ts

```diff
@@ -0,0 +1,57 @@
+import * as _ from 'lodash'
+
+const equality = (from:string, to:string) => ['Equals', from, to]
+
+const TypeUtils = {
+	nativeFactTypeTemplates: {
+		equality: {
+			  'is equal to': equality
+			, 'equals': equality
+		} ,
+		comparison: {
+			'is greater than': (from:string, to:string) => ['GreaterThan', from, to]
+			, 'is greater than or equal to': (from:string, to:string) => ['GreaterThanOrEqual', from, to]
+			, 'is less than': (from:string, to:string) => ['LessThan', from, to]
+			, 'is less than or equal to': (from:string, to:string) => ['LessThanOrEqual', from, to]
+			, 'is equal to': equality
+			, 'equals': equality
+		}
+	},
+
+	validate: {
+		integer: (value:any, required:boolean, callback:Callback<number>) => {
+			let processedValue = parseInt(value, 10)
+			if(_.isNaN(processedValue)) {
+				callback('is not a number: ' + value)
```

Comment:
It would be a breaking change, but it's one we need to make eventually - I'd keep it as a separate PR though